definitions/npm/styled-components_v2.x.x/flow_v0.53.x-/styled-components_v2.x.x.js

```diff
@@ -0,0 +1,369 @@
+  path:                     $npm$styledComponents$StyledComponentsComponentListValue,
+  pattern:                  $npm$styledComponents$StyledComponentsComponentListValue,
+  polygon:                  $npm$styledComponents$StyledComponentsComponentListValue,
+  polyline:                 $npm$styledComponents$StyledComponentsComponentListValue,
+  radialGradient:           $npm$styledComponents$StyledComponentsComponentListValue,
+  rect:                     $npm$styledComponents$StyledComponentsComponentListValue,
+  stop:                     $npm$styledComponents$StyledComponentsComponentListValue,
+  svg:                      $npm$styledComponents$StyledComponentsComponentListValue,
+  text:                     $npm$styledComponents$StyledComponentsComponentListValue,
+  tspan:                    $npm$styledComponents$StyledComponentsComponentListValue,
+|}
+
+declare module 'styled-components' {
+  declare type Interpolation                                              = $npm$styledComponents$Interpolation;
+  declare type NameGenerator                                              = $npm$styledComponents$NameGenerator;
+  declare type Theme                                                      = $npm$styledComponents$Theme;
+  declare type ThemeProviderProps                                         = $npm$styledComponents$ThemeProviderProps;
+  declare type TaggedTemplateLiteral<R>                                   = $npm$styledComponents$TaggedTemplateLiteral<R>;
+
+  declare type ReactComponentFunctional<Props: {}, DefaultProps: ?{} = *> = $npm$styledComponents$ReactComponentFunctional<Props, DefaultProps>;
+  declare type ReactComponentFunctionalUndefinedDefaultProps<Props: {}>   = $npm$styledComponents$ReactComponentFunctionalUndefinedDefaultProps<Props>;
+  declare type ReactComponentClass<Props: {}, DefaultProps: ?{} = *>      = $npm$styledComponents$ReactComponentClass<Props, DefaultProps>;
+  declare type ReactComponentClassUndefinedDefaultProps<Props: {}>        = $npm$styledComponents$ReactComponentClassUndefinedDefaultProps<Props>;
+  declare type ReactComponentUnion<Props>                                 = $npm$styledComponents$ReactComponentUnion<Props>;
+  declare type ReactComponentIntersection<Props>                          = $npm$styledComponents$ReactComponentIntersection<Props>;
+  declare type ReactComponentStyledStaticProps<Props>                     = $npm$styledComponents$ReactComponentStyledStaticProps<Props, $Subtype<$Keys<$npm$styledComponents$StyledComponentsComponentList<*>>>>;
+  declare type ReactComponentStyled<Props>                                = $npm$styledComponents$ReactComponentStyled<Props, $Subtype<$Keys<$npm$styledComponents$StyledComponentsComponentList<*>>>>;
+  declare type ReactComponentStyledTaggedTemplateLiteral<Props>           = $npm$styledComponents$ReactComponentStyledTaggedTemplateLiteral<Props, $Subtype<$Keys<$npm$styledComponents$StyledComponentsComponentList<*>>>>;
```

Comment:
> $npm$styledComponents$StyledComponentsComponentList<*>

`$npm$styledComponents$StyledComponentsComponentList` doesn't take a type argument.

Reply:
Ah, you're right. Missed updating that...

Reply:
Or, better yet, it could just use ` $npm$styledComponents$StyledComponentsComponentListKeys`.